### ISSUE TITLE
when train at 10th epoch,it suddenly end

### ISSUE THREAD

**Issue by hexin6688** [qqwweee/keras-yolo3#657]
Epoch 9/50
252/252 [==============================] - 386s 2s/step - loss: 346.2141 - val_loss: nan
Epoch 10/50
251/252 [============================>.] - ETA: 1s - loss: 286.6874
Process finished with exit code -1
anyone have faced this problem?

**Comment by GiangHLe** [comment]
In train.py, the author used early stopping as callback.

https://github.com/qqwweee/keras-yolo3/blob/e6598d13c703029b2686bc2eb8d5c09badf42992/train.py#L39

They monitor the training process by validation loss, just change or comment on the early_stopping parameter inline 84, you will be fine.

https://github.com/qqwweee/keras-yolo3/blob/e6598d13c703029b2686bc2eb8d5c09badf42992/train.py#L84

**Comment by 18875140516** [comment]
> 286.6874

hello, have you finished your training? how much loss is your model? When I test my model which loss down to nearly 100 after 40epoches, it can't find the ground truth.

**Comment by hexin6688** [comment]
maybe the loss is too high that cause you can't find the groundtruth.maybe you can add epoches,and decrease the loss under 10 @

> > 286.6874
> 
> hello, have you finished your training? how much loss is your model? When I test my model which loss down to nearly 100 after 40epoches, it can't find the ground truth.
